webservices/config.py

```diff
@@ -32,7 +32,6 @@ def get_cycle_end(year):
 REQUIRED_CREDS = (
     'SQLA_CONN',
     'FEC_SLACK_TOKEN',
-    'MANDRILL_API_KEY',
```

Comment:
Do we need to add the `SLACK_HOOK` environment variable here in addition to the `FEC_SLACK_TOKEN` that's already here, or no?

Reply:
I thought about it but, I would not want not having the slack token to cause the build to fail. I assume we will notice that it is broken.